pages/fundamentals/bounds-check-elimination.html

```diff
@@ -215,7 +215,7 @@ <h3>Example 3</h3>
 	index := 8
 
 	// In Go, for the subslice syntax s[a:b],
-	// the relations 0 <= a <= b <= cap(s) must
+	// the relations 0 <= a <= b <= len(s) must

```

Comment:
@i6u 
I think the current description is correct here.
And it is what is emphasized in the code.